daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-dapr-extension.md

```diff
@@ -0,0 +1,77 @@
+---
+type: docs
+title: "Dapr Visual Studio Code extension overview"
+linkTitle: "Dapr extension"
+weight: 10000
+description:  "How to develop and run Dapr applications with the Dapr extension"
+---
+
+
+Dapr offers a *preview* [Dapr Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-dapr) for local development and debugging of your Dapr applications.
+
+<a href="vscode:extension/ms-azuretools.vscode-dapr" class="btn btn-primary" role="button">Open in VSCode</a>
+
+## Features
+
+### Scaffold Dapr debugging tasks
+
+The Dapr extension helps you debug your applications with Dapr using Visual Studio Code's [built-in debugging capability](https://code.visualstudio.com/Docs/editor/debugging).
+
+Using the `Dapr: Scaffold Dapr Tasks` [Command Palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) operation, you can update your existing `task.json` and `launch.json` files to launch and configure the Dapr sidecar when you begin debugging.
+
+1. Make sure you have a launch configuration set for your app. ([Learn more](https://code.visualstudio.com/Docs/editor/debugging))
+1. Open the Command Palette with `Ctrl+Shift+P`
+1. Select `Dapr: Scaffold Dapr Tasks`
+1. Run your app and the Dapr sidecar with `F5` or via the Run view.
+
+### Scaffold Dapr components
+
```

Comment:
Is this the only new feature added? I thought that there were others? Will these be in another PR and why have just this one and not say the dashboard button? Is only this one finished?

Reply:
This was actually not a new feature but a feature that just wasn't captured in the older version as it was made by Phillip in the older versions of Dapr which didn't organize the Components folder automatically for you I believe. The newer features from 0.5 update aren't capture because I didn't want to add a feature I couldn't guarantee would be out when this document would be.

Reply:
There are new features already implemented but they just aren't in the latest release right now.